R/DummyFeaturesWrapper.R

```diff
@@ -0,0 +1,35 @@
+#' @title Fuse learner with dummy feature creator.
+#'
+#' @description
+#' Fuses a base learner with the dummy feature creator (see \code{\link{createDummyFeatures}}).
+#' Returns a learner which can be used like any other learner.
+#'
+#' @template arg_learner
+#' @inheritParams createDummyFeatures
+#' @template ret_learner
+#' @family wrapper
+#' @export
+makeDummyFeaturesWrapper = function (learner, method = "1-of-n", cols = NULL) {
+  learner = checkLearner(learner)
+  args = list(method = method, cols = cols)
+  rm(list = names(args))
```

Comment:
We are using this also in other wrappers (e.g. impute wrapper). 
I talked to @mllg and @berndbischl at the last workshop because I also found this confusing. They explained why this makes sense, but I forgot the reason...